superset-frontend/src/dashboard/components/gridComponents/Chart.jsx

```diff
@@ -175,6 +180,11 @@ export default class Chart extends React.Component {
         this.resizeTimeout = setTimeout(this.resize, RESIZE_TIMEOUT);
       }
 
+      if (nextState.isUpdated) {
+        this.setState({ isUpdated: false });
```

Comment:
It is not recommended to call `setState` in SCU, it is rather hacky. If you want to change state when a prop change, please consider using [static getDerivedStateFromProps](https://reactjs.org/docs/react-component.html#static-getderivedstatefromprops).